jb/testing/tde_square_sum.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef jb_testing_tde_square_sum_hpp
+#define jb_testing_tde_square_sum_hpp
+
+#include <jb/fftw/tde_result.hpp>
+
+namespace jb {
+namespace testing {
+
+/**
+ * Compute the sum square of a family of timeseries.
+ *
+ * @return tde_result with the square sum of each timeseries
+ *
+ * @tparam container_t type of the collection of timeseries
+ */
+template <typename container_t>
+auto sum_square(container_t const& ts) {
```

Comment:
ACK, I think sum_square explains what is intended, so I will change the file name